repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/content-cards.ftl

```diff
@@ -1,12 +1,15 @@
 <#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"] >
 <#import "./link.ftl" as hlink>
 
-<#macro contentCards contentCards>
+<#macro contentCards contentCards size="third">
     <#if contentCards.header?has_content>
         <div class="nhsuk-u-reading-width">
             <h2>${contentCards.header}</h2>
         </div>
     </#if>
+    <#if contentCards.cardGroupSummary?has_content>
+        <p>${contentCards.cardGroupSummary}</p>
```

Comment:
Since `cardGroupSummary` is a TextArea, the line breaks aren't automatically translated to `<br>` tags. Either we need to perform this translation like `${contentCards.cardGroupSummary?replace('\n', '<br>')}` or we can change `cardGroupSummary` as a RichText field.

Reply:
done